src/helpers/cache.ts

```diff
@@ -1,20 +1,40 @@
-import { join } from 'path'
+import { existsSync } from 'fs'
+import { join, resolve } from 'path'
 
-export const restoreCache = async ({ cache, publish }) => {
-  const cacheDir = join(publish, 'cache')
+import { shouldSkip } from './utils'
+
+const findDistDir = (publish) => {
+  if (!shouldSkip()) {
+    return publish
+  }
+  // In this situation, the user has disabled the plugin, which means that they might be using next export,
+  // so we'll look in a few places to finx the site root
+  for (const root of [resolve(publish, '..'), resolve(publish, '..', '..')]) {
+    if (existsSync(join(root, 'next.config.js'))) {
+      return join(root, '.next')
+    }
+  }
+  return null
+}
 
-  if (await cache.restore(cacheDir)) {
+export const restoreCache = async ({ cache, publish }) => {
+  const distDir = findDistDir(publish)
+  if (!distDir) {
+    return
+  }
+  if (await cache.restore(join(distDir, 'cache'))) {
     console.log('Next.js cache restored.')
   } else {
     console.log('No Next.js cache to restore.')
   }
 }
 
 export const saveCache = async ({ cache, publish }) => {
-  const cacheDir = join(publish, 'cache')
-
-  const buildManifest = join(publish, 'build-manifest.json')
```

Comment:
Just to confirm: was removing `build-manifest.json` as the `digests` option intentional?

Reply:
:+1: